.github/pull_request_template.md

```diff
@@ -1,10 +1,12 @@
 # Motivation
-A jira link to the [ticket](https://katanaos.atlassian.net/browse/[ticket-number])
+
+Short explanation on why this change is needed and reference to ticket (link created automatically): TICKET-123
```

Comment:
@allanpaiste The links are created automatically?

Reply:
yes, for every repo the rules are defined in https://github.com/katana-scm/katana-ui/settings/key_links